poc_iot_verifier/src/poc.rs

```diff
@@ -228,28 +228,13 @@ impl Poc {
             }
         };
         tracing::debug!("witness info {:?}", beaconer_info);
-
-        // calculate distance of witness to beaconer
-        let witness_location = witness_info
-            .location
-            .ok_or(VerificationError::NotFound("invalid witness_location"))?;
-        let beacon_location = beaconer_info
-            .location
-            .ok_or(VerificationError::NotFound("invalid beaconer_location"))?;
-        let witness_distance = calc_distance(beacon_location, witness_location)?;
-
         // run the witness verifications
         match self
-            .do_witness_verifications(
-                &witness_info,
-                witness_report,
-                beaconer_info,
-                witness_distance,
-            )
+            .do_witness_verifications(&witness_info, witness_report, beaconer_info)
             .await
         {
             Ok(()) => {
-                if let Some(hex_scale) = hex_density_map.get(witness_location).await {
+                if let Some(hex_scale) = hex_density_map.get(witness_info.location.unwrap()).await {
```

Comment:
Don't we need to handle this `unwrap` better?

Reply:
so at this point the location has already been verified and thus we know unwrap will work.

i did attempt to not use unwrap like this but i end up doing something elaborate and it feels like just for the sake of it.

Open to suggestions